cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/leveldb/LevelDBQueueAdmin.java

```diff
@@ -148,8 +148,28 @@ public String getActualTableName(QueueName queueName) {
                                             queueName.getThirdComponent()));
   }
 
+  public TableId getDataTableId(Id.Flow flowId) {
+    // tableName = system.queue.<app>.<flow>
+    return TableId.from(flowId.getNamespaceId(), getDataTableName(flowId));
```

Comment:
hmmm, for leveldb we still need to go directly to ``TableId``. That perhaps answers my previous question - so we need to make sure that only hbase classes never create a ``TableId`` directly. Not for this PR, but unsure if there is some scope for refactoring here then.